it/pages/TwoEyeReviewerControllerISpec.scala

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package pages
+
+import models.errors.{BadRequestError, InternalServerError, NotFoundError, StaleDataError}
+import play.api.http.Status
+import play.api.libs.json.{JsValue, Json}
+import play.api.libs.ws.{WSRequest, WSResponse}
+import stubs.{AuditStub, AuthStub, ExternalGuidanceStub}
+import support.IntegrationSpec
+
+class TwoEyeReviewerControllerISpec extends IntegrationSpec {
+
+  private val successPayload: JsValue = Json.parse(
+    """{
+      |  "id":"oct90005",
+      |  "title":"Telling HMRC about extra income",
+      |  "lastUpdated":"2020-05-10",
+      |  "pages":[
+      |    {
+      |      "id":"id1",
+      |      "title":"how-did-you-earn-extra-income",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id2",
+      |      "title":"sold-goods-or-services/did-you-only-sell-personal-possessions",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id3",
+      |      "title":"sold-goods-or-services/have-you-made-a-profit-of-6000-or-more",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id4",
+      |      "title":"sold-goods-or-services/have-you-made-1000-or-more",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id5",
+      |      "title":"sold-goods-or-services/you-do-not-need-to-tell-hmrc",
+      |      "status":"NotStarted"}
+      |    ,
+      |    {
+      |      "id":"id6",
+      |      "title":"rent-a-property/do-you-receive-any-income",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id7",
+      |      "title":"rent-a-property/have-you-rented-out-a-room",
+      |      "status":"NotStarted"
+      |    }
+      |  ]
+      |}""".stripMargin
+  )
+
+  "Calling the approval 2i review endpoint" should {
+
+    "return an Ok response" in {
```

Comment:
could add "when the id passed is found" to make it clear what is being tested

Reply:
I have updated the test description to make the intention of the test more clear.